universe/Universe.h

```diff
@@ -520,6 +523,7 @@ class FO_COMMON_API Universe {
 
     std::map<std::string, std::map<int, std::map<int, double>>>
                                     m_stat_records;                     ///< storage for statistics calculated for empires. Indexed by stat name (string), contains a map indexed by empire id, contains a map from turn number (int) to stat value (double).
+    bool                            m_stat_records_publish;             ///< send empire statistics to the players.
```

Comment:
is this variable beneficial vs. just polling the OptionsDB each time?

Reply:
`OptionsDB` isn't included in `SerializeUniverse.cpp`

Reply:
I'm hesitant to add it as a member of Universe... seems like it should be a parameter to the function (though that's probably not possible in practice) or stored elsewhere. That said, there is already `m_encoding_empire`

Reply:
Does `boost::serialize` have something like context?

Reply:
Don't know... Is there a reason to not include OptionsDB in SerializeUniverse?

Reply:
I suppose it possible to transfer additional parameters in derived `Archive` class: https://www.boost.org/doc/libs/1_58_0/libs/serialization/doc/derivation.html
But it will definitely require amount of refactoring.

Reply:
Seems unnecessary...

Reply:
Added OptionsDB instead of `m_stat_records_publish`.